Source/Scene/Model.js

```diff
@@ -3475,6 +3476,21 @@ define([
         }
     }
 
+    function checkSupportedGlExtensions(model, context) {
+        var glExtensionsUsed = model.gltf.glExtensionsUsed;
+        if (defined(glExtensionsUsed)) {
+            var glExtensionsUsedLength = glExtensionsUsed.length;
+            for (var i = 0; i < glExtensionsUsedLength; i++) {
+                var extension = glExtensionsUsed[i];
+                if (extension !== 'OES_element_index_uint') {
+                    throw new RuntimeError('Unsupported WebGL Extension: ' + extension);
+                } else if (!context.elementIndexUint) {
+                    throw new RuntimeError('OES_element_index_uint WebGL extension is not enabled.');
```

Comment:
Perhaps `'The OES_element_index_uint WebGL extension is required for this glTF model, but it is not supported by this system'`